docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -199,75 +261,59 @@ What's interesting is that the transaction now has 3 account updates:
 
-```ts
-class MyContract extends SmartContract {
-  @method payout(amount: UInt64) {
-    // this.send({ to: this.sender, amount }); // commented this out, placed it below
-  }
-}
+<!-- TODO: _ideal_ here would be a little picture of a tree, with a parent and a child node -->
 
-// ...
+Concretely, in this example, both the zkApp account update and the user account update are public input to the zkApp method call. 
 
-const tx = await Mina.transaction(sender, () => {
-  let amount = UInt64.from(5n * MINA);
-  zkapp.payout(amount);
-  zkapp.send({ to: sender, amount }); // here
-});
-await tx.prove();
-```
+Intuitively, the public input means that the zkApp can "see" and constrain the update as part of its proof. Here, it means that no one can change the public key of the receiver, or amount they receive, without making the proof invalid. 
 
-If you run this, you'll see an error:
+The update can contain only what the method specified.
 
-```
-Error when proving MyContract.payout()
-...
-```
+All of this is true because `this.send()` placed the receiver update at call depth 1, under the zkApp update. 
 
-This error comes about as follows:
+As a counter example: The fee payer is never part of the public input. It can be anything without affecting the validity of the proof.
 
-- The logic that's proven is what's contained in your method -- `payout()`
-- The method doesn't contain the `this.send()` command, which is therefore not reflected in the account updates the method creates
-- The actual account updates in your transaction do reflect the `send()` command
-- The actual account updates are passed into your method prover as public input
-- The method prover compares that public input to the account updates it itself creates (which makes them sort of a "public output")
-- That comparison check fails, because the two sets of account updates are different
--->
+A key takeaway is: If you want something to become part of your proof, you must put it inside your `@method`.
 
 ## Signing transactions and explicit account updates
 
-Let's recap: We have explained how to write a SmartContract. We've seen how to create a transaction which calls that contract, and how the transaction consists of account updates which were created by o1js under the hood. Now, we'll see an example of creating an account update explicitly. We'll also learn how to use signatures, for authorizing updates to user accounts.
+To recap the workflow covered so far: You write a smart contract, and then create a transaction to call the smart contract. The workflow of this transaction consists of account updates that are created by o1js. 
+
+The next example explicitly creates an account update and shows how to use signatures for authorizing updates to user accounts.
+
+To continue the [Payment to zkApp example](#payment-to-zkapp-example) in the other direction: make a deposit from the user into the zkApp. 
```

Comment:
Right, as soon as we focus the content on what the user is doing, we know to speak to them directly "you" (because we are not there with them). It's a thing.